cmd/news/news.go

```diff
@@ -0,0 +1,78 @@
+package news
+
+/*
+Practise go by creating a function that retrieves an the information from the following BBC News xml link and gives a list of all the current headlines.
+http://feeds.bbci.co.uk/news/technology/rss.xml
+*/
+
+import (
+	"encoding/xml"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	url = "http://feeds.bbci.co.uk/news/technology/rss.xml"
```

Comment:
extend the functionality to take the url in as an command line argument.

additional test feeds: 

- https://www.nasa.gov/rss/dyn/breaking_news.rss
- https://rss.nytimes.com/services/xml/rss/nyt/US.xml
- https://lifehacker.com/rss